lib/wovnrb/headers.rb

```diff
@@ -115,11 +110,11 @@ def redirect_location(lang)
         case @settings['url_pattern']
         when 'query'
           lang_param_name = @settings['lang_param_name']
-          if location !~ /\?/
-            location = "#{location}?#{lang_param_name}=#{lang_code}"
-          else @env['REQUEST_URI'] !~ /(\?|&)#{lang_param_name}=/
-               location = "#{location}&#{lang_param_name}=#{lang_code}"
-          end
+          location = if location =~ /\?/
+                       "#{location}&#{lang_param_name}=#{lang_code}"
+                     else
```

Comment:
This code is just trying to append the query with either `?` or `#`. The `@env['REQUEST_URI'] !~ /(\?|&)#{lang_param_name}=/` expression was unused.